src/sidebar/components/tag-editor.js

```diff
@@ -67,7 +67,7 @@ function TagEditor({ onEditTags, tags: tagsService, tagList }) {
       // Remove any repeated suggestions that are already tags
       // and set those to state.
       setSuggestions(removeDuplicates(suggestions, tagList));
-      setSuggestionsListOpen(true);
+      setSuggestionsListOpen(suggestions.length > 0);
```

Comment:
This probably should have been this way in the first place, but does not have any apparent effect because the `<Autocomplete>` list also has a check on length and won't render. But doing this correctly here then allows us to leverage this variable for the new condition below. Previous it was still true when the list was empty because we were not updating it. Now it is correctly updated.